docs/en/guides/testing-async-components.md

```diff
@@ -0,0 +1,159 @@
+# Testing Asynchronous Components
```

Comment:
This headline might be confused with ["Async components"](https://vuejs.org/v2/guide/components.html#Async-Components).

Maybe: "Testing Asynchronous Behaviour" ?

Reply:
Good catch. I think your suggested title is better. Mintor, but In my day to day life I spell it behaviour, should use use American English (no u) or English/Europe/everywhere else English?

Reply:
Hm. I think we should probably stick do American english as that's the bigger "market"?

OTOH @eddyerburgh is british so ..., :-P

Reply:
Indeed I am British 🇬🇧

Reply:
I am Australian (English colony) so we use the same style. But I agree we should use the American English since I believe the official docs would use that.

Reply:
Yeah, we should use US English